src/main/resources/anniv6Resources/localization/eng/Grass/UIstrings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "${ModID}:Grass": {
+    "TEXT": [
+      "Grass",
+      "The Spire has grass. Grass needs Energy to turn into juicy vegetables.",
```

Comment:
I just merged in code that will show the author for each zone. You should add an entry here with the name you'd like to be credited by. You can pull and look at other zones for examples.

Since you have other text here you'll probably need to adjust the code where they're used since inserting an entry here will shift the indexes, sorry about that!

Also, you should add yourself with the same name to ModTheSpire.json.